java/google/registry/rde/imports/XjcToContactResourceConverter.java

```diff
@@ -90,6 +98,23 @@ static ContactResource convertContact(XjcRdeContact contact) {
         .build();
   }
 
+  private static HistoryEntry createHistoryEntry(XjcRdeContact contact) {
+    XjcRdeContactElement element = new XjcRdeContactElement(contact);
+    final HistoryEntry historyEntry =
+        new HistoryEntry.Builder()
+            .setType(HistoryEntry.Type.RDE_IMPORT)
+            .setClientId(contact.getClID())
+            .setTrid(Trid.create(null))
+            .setModificationTime(DateTime.now())
```

Comment:
DateTime.now() is a bad idea. Use SystemClock.nowUtc() for this moment, but in actuality I think you want a transaction-frozen time to get correct ordering of commit logs. Accept a DateTime as a parameter to this, and call it with ofy().getTransactionTime() within transactions, or do the ofy().getTransactionTime() call here and add a checkState(ofy().isInTransaction());